### PR TITLE
Update operations guide based on Jakarta EE 8 retrospective

### PR DESCRIPTION
**Pull request by starksm64**
Signed-off-by: starksm64 <starksm64@gmail.com>

### CODE REVIEW

**Comment by starksm64** [comment]
Look into incorporating the spec Jakatifcation document https://docs.google.com/document/d/1cARrgRBSAeMkZcUHxrrMqCGh84reZBtcJRcH9dyZeWo/edit

File: operations.adoc [hunk 1]
```diff
@@ -70,18 +71,33 @@ For each final specification:
...
-There is no need to produce a final EPL version?
-Question on issues with using mirrored links.
-4. Attached the EFTL TCK summary results for each implementation included with this release review.
-5. Create two https://help.github.com/en/articles/about-pull-requests#draft-pull-requests[draft PRs] request against the
+1. Produced a staging release via OSSRH staging repository for the api, javadoc.
+** Javadocs should include the https://raw.githubusercontent.com/eclipse-ee4j/jakartaee-api/master/licenses/EFSL.html[Eclipse Foundation Specification License].
+** Version in pom.xml should be increased by comparing to the previously released API artifact.
+** Existing "@version ..." JavaDoc tags should be updated to match, or removed.
+1. The candidate final https://www.eclipse.org/legal/tck.php[EFTL] licensed TCK archive should be uploaded to the project directory under http://download.eclipse.org/ee4j, e.g., http://download.eclipse.org/ee4j/bean-validation/beanvalidation-tck-dist-2.0.5.zip .
+1. Generate standalone TCK results or platform TCK result as apporiate for the spec project.
```
Comment (kwsutter) [inline operations.adoc]: manual spellchecker again...  :-)  ..appropriate..

File: operations.adoc [hunk 2]
```diff
@@ -70,18 +71,33 @@ For each final specification:
...
-4. Attached the EFTL TCK summary results for each implementation included with this release review.
-5. Create two https://help.github.com/en/articles/about-pull-requests#draft-pull-requests[draft PRs] request against the
+1. Produced a staging release via OSSRH staging repository for the api, javadoc.
+** Javadocs should include the https://raw.githubusercontent.com/eclipse-ee4j/jakartaee-api/master/licenses/EFSL.html[Eclipse Foundation Specification License].
+** Version in pom.xml should be increased by comparing to the previously released API artifact.
+** Existing "@version ..." JavaDoc tags should be updated to match, or removed.
+1. The candidate final https://www.eclipse.org/legal/tck.php[EFTL] licensed TCK archive should be uploaded to the project directory under http://download.eclipse.org/ee4j, e.g., http://download.eclipse.org/ee4j/bean-validation/beanvalidation-tck-dist-2.0.5.zip .
+1. Generate standalone TCK results or platform TCK result as apporiate for the spec project. 
+** Information on running TCKs in the Jakarta CI infrastructure can be found https://wiki.eclipse.org/TCK:Build_From_Jakarta_EE_TCK_Repo_And_Run#Jenkins_Pipelines[here].
```
Comment (kwsutter) [inline operations.adoc]: I personally don't like the "here" hotlinks.  They are too easy to miss.  I'd prefer something like "can be found in the 'url'[Jakarta Jenkins documentation]".  But, this is just a nit...

File: operations.adoc [hunk 3]
```diff
@@ -70,18 +71,33 @@ For each final specification:
...
+*** A subdirectory major.minor corresponding to the version of the spec, (e.g., 1.6), that contains:
+**** Specification Document from (2) above in both pdf and html formats, e.g., wombat_1.6.pdf and wombat_1.6.html
+**** Summary results of TCK run showing at least one compatible implementation
+**** Link to final TCK test bundle if the spec defines a TCK. This will be signed and uploaded to the offical
+specification download area when the ballot passes.
+**** The URL of the OSSRH staging repository for the api, javadoc artifacts
+**** An apidocs directory containing the final JavaDocs from the api build in the second PR.
+1. Update the Jakarta EE API jar by submitting a PR to the https://github.com/eclipse-ee4j/jakartaee-api[jakartaee-api] project that updates the version number of your API jar file.
+1. Update Eclipse GlassFish to use the new version of your API (and implementation, if applicable) by submitting a PR to https://github.com/eclipse-ee4j/glassfish[GlassFish].
+1. For any release other than a service release, create a release record as described in https://www.eclipse.org/projects/handbook/#pmi-commands-release and then:
```
Comment (kwsutter) [inline operations.adoc]: On today's Spec call, we determined that we should create these Eclipse Release Records as soon as we realize one is needed. Does this doc talk about the Planning process?  Maybe we could add something there?

For this particular bullet, maybe just change it to "create a release record (if one doesn't already exist) as described.."

File: operations.adoc [hunk 4]
```diff
@@ -70,18 +71,33 @@ For each final specification:
...
+** Request approval for the release from the PMC by sending an email to ee4j-pmc@eclipse.org referencing the release record.
+** Deliver an IP Log to the IP Team for their review as described in https://www.eclipse.org/projects/handbook/#pmi-commands-iplog.
+** Contact the EMO to initiate the release review by sending an email to emo@eclipse.org.
+1. When the ballot request PRs are approved, release staged artifacts to Maven Central. Advice on this can be found https://wiki.eclipse.org/MavenReleaseScript[here].
```
Comment (kwsutter) [inline operations.adoc]: Similar comment on the use of [here] hotspots.

**Pull request merged**